### PR TITLE
Pf/pylint fix

### PR DESCRIPTION
**Pull request by pfinashx**
Fixed Pylint warnings

### CODE REVIEW

**Comment by AlexKoff88** [comment]
@ljaljushkin, can you please take a look?

File: nncf/quantization/metrics.py [hunk 1]
```diff
@@ -21,6 +22,8 @@ def get_metric_table(self):
         pass
 
 
+# pylint:disable=protected-access
+# pylint:disable=unused-variable
```
Comment (ljaljushkin) [inline nncf/quantization/metrics.py:26]: Wwhere were this two warning reported? Is it really False Positive?

File: nncf/quantization/metrics.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from texttable import Texttable
 from collections import deque
 
+# pylint:disable=unused-import
```
Comment (ljaljushkin) [inline nncf/quantization/metrics.py:7]: Have you tried to delete `get_all_modules_by_type`? Looks like it's better to remove it this import?

File: nncf/pruning/filter_pruning/algo.py [hunk 3]
```diff
@@ -29,6 +29,7 @@
 from nncf.pruning.utils import get_rounded_pruned_element_number
 
 
+# pylint:disable=useless-super-delegation
 @COMPRESSION_ALGORITHMS.register('filter_pruning')
 class FilterPruningBuilder(BasePruningAlgoBuilder):
     def __init__(self, config, should_init: bool = True):
```
Comment (ljaljushkin) [inline nncf/pruning/filter_pruning/algo.py:35]: should be resolved by deleting `__init__` method

File: nncf/initialization.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+# pylint:disable=unused-import
```
Comment (ljaljushkin) [inline nncf/initialization.py:1]: delete unused `import warnings`

File: nncf/quantization/metrics.py (hunk 1 above)
Comment (pfinashx) [reply nncf/quantization/metrics.py:26]: This is not my code, so decided not delete unused variables, perhaps they are imported in other files. 
Pylint mark variables on 255, 361 and 498 lines as unusable.
Protected-access warnings are in 295, 362, 365, 419, 429, 458, 480 lines

File: nncf/quantization/metrics.py (hunk 2 above)
Comment (pfinashx) [reply nncf/quantization/metrics.py:7]: Same here, I decided not delete unused import - from nncf.utils import get_all_modules_by_type on 9 line

**Review by ljaljushkin — CHANGES_REQUESTED** [review]
please avoid introducing comments `#pylint: disable` as much as possible. I've spotted several places where this comment isn't necessary

File: nncf/quantization/metrics.py (hunk 1 above)
Comment (ljaljushkin) [reply nncf/quantization/metrics.py:26]: please remove these variables, find author by git blame and add them to review.
AFAIK, @asenina was working on this feature.

File: nncf/quantization/metrics.py (hunk 2 above)
Comment (ljaljushkin) [reply nncf/quantization/metrics.py:7]: It shouldn't impact the functionality. But still it worth mentioning the author

File: nncf/quantization/metrics.py (hunk 1 above)
Comment (asenina) [reply nncf/quantization/metrics.py:26]: @pfinashx  Please take changes for `nncf\quantization\metrics.py` on this [branch](https://github.com/asenina/nncf_pytorch/tree/as/fix_pylint) . Thanks.

Comment (pfinashx) [reply nncf/quantization/metrics.py]: @asenina Done, thanks

File: nncf/quantization/metrics.py (hunk 2 above)
Comment (pfinashx) [reply nncf/quantization/metrics.py]: Done

File: nncf/pruning/filter_pruning/algo.py (hunk 3 above)
Comment (pfinashx) [reply nncf/pruning/filter_pruning/algo.py]: Done

File: nncf/initialization.py (hunk 4 above)
Comment (pfinashx) [reply nncf/initialization.py]: Done

**Comment by pfinashx** [comment]
Waiting Jenkins with Pylint check and precommit.

**Pull request merged**